splice_machine/src/main/java/com/splicemachine/derby/utils/StatisticsOperation.java

```diff
@@ -58,7 +62,11 @@
     protected ScanSetBuilder scanSetBuilder;
     protected String scope;
 
-    public StatisticsOperation(ScanSetBuilder scanSetBuilder, String scope, Activation activation) {
+    // serialization
+    public StatisticsOperation(){}
+
+    public StatisticsOperation(ScanSetBuilder scanSetBuilder, String scope, Activation activation) throws StandardException {
+        super(new DerbyOperationInformation(activation, 0, 0, 0));
```

Comment:
Is it better to use super(activation, 0, 0, 0)? 
It seems that this way, we also set the two variables optimizerEstimatedRowCount and optimizerEstimatedCost.
Otherwise, it is all good for me.